code/game/machinery/vending/marine_vending.dm

```diff
@@ -200,24 +201,23 @@
 	contraband = list(
 					/obj/item/ammo_magazine/smg/ppsh/ = 20,
 					/obj/item/ammo_magazine/smg/ppsh/extended = 4,
-					/obj/item/ammo_magazine/sniper = 0,
-					/obj/item/ammo_magazine/sniper/incendiary = 0,
-					/obj/item/ammo_magazine/sniper/flak = 0,
-					/obj/item/ammo_magazine/rifle/m4ra = 0,
-					/obj/item/ammo_magazine/rifle/incendiary = 0,
-					/obj/item/ammo_magazine/rifle/m41aMK1 = 0,
-					/obj/item/ammo_magazine/rifle/lmg = 0,
-					/obj/item/ammo_magazine/pistol/hp = 0,
-					/obj/item/ammo_magazine/pistol/heavy = 0,
-					/obj/item/ammo_magazine/pistol/holdout = 0,
-					/obj/item/ammo_magazine/pistol/highpower = 0,
-					/obj/item/ammo_magazine/pistol/vp70 = 0,
-					/obj/item/ammo_magazine/revolver/small = 0,
-					/obj/item/ammo_magazine/revolver/cmb = 0,
-					/obj/item/ammo_magazine/smg/mp7 = 0,
-					/obj/item/ammo_magazine/smg/skorpion = 0,
-					/obj/item/ammo_magazine/smg/uzi = 0,
-					/obj/item/ammo_magazine/smg/p90 = 0
+					/obj/item/ammo_magazine/pistol/holdout = 12
+					// /obj/item/ammo_magazine/sniper = 0,
+					// /obj/item/ammo_magazine/sniper/incendiary = 0,
+					// /obj/item/ammo_magazine/sniper/flak = 0,
+					// /obj/item/ammo_magazine/rifle/m4ra = 0,
+					// /obj/item/ammo_magazine/rifle/incendiary = 0,
+					// /obj/item/ammo_magazine/rifle/m41aMK1 = 0,
+					// /obj/item/ammo_magazine/rifle/lmg = 0,
+					// /obj/item/ammo_magazine/pistol/heavy = 0,
+					// /obj/item/ammo_magazine/pistol/highpower = 0,
+					// /obj/item/ammo_magazine/pistol/vp70 = 0,
+					// /obj/item/ammo_magazine/revolver/small = 0,
+					// /obj/item/ammo_magazine/revolver/cmb = 0,
+					// /obj/item/ammo_magazine/smg/mp7 = 0,
+					// /obj/item/ammo_magazine/smg/skorpion = 0,
+					// /obj/item/ammo_magazine/smg/uzi = 0,
+					// /obj/item/ammo_magazine/smg/p90 = 0
```

Comment:
We already decided to leave this around, it's for stocking bought contraband ammo and for events.

Reply:
Oh whoops.